src/UserFrontend.php

```diff
@@ -61,10 +64,26 @@ public static function login_head() {
    */
   public static function login_form_defaults(array $args) {
     $args['value_remember'] = TRUE;
+    $args['remember'] = FALSE;
     return $args;
   }
 
   /**
+   * @implements login_form
+   */
+  public static function login_form() {
+    ob_start(__CLASS__ . '::removesForgetmenotCheckbox');
```

Comment:
@sun — ok. I supposed, as said in the description, that remove the output element could be better that hide it via CSS. However, if you prefer this option: let's do that. 

Fixup commit on the way. 🙂